products/AOS-CX-Switching

```diff
@@ -0,0 +1,210 @@
+---
+title: Aruba AOS-CX-Switching
+layout: post
+# Possible values are os,db,app,lang,framework,device,service,server-app
+# If you add a new value, please mention it on the PR Description. Some rough guidelines:
+# os is operating systems (and similar projects)
+# db is databases, os is operating systems, lang is programming languages
+# app is end-user applications
+# lang is programming languages
+# framework is used for application libraries, SDKs, frameworks etc
+# device is physical devices
+# service is managed service offerings (SaaS/PaaS etc)
+# server-app are applications usually installed on the server-side
+category: device
+
+# What should be used to sort releases. Set to one of:
+# releaseCycle/eol/support/release/cycleShortHand/latest/latestShortHand
+# which must be present in the releases underneath
+sortReleasesBy: "releaseCycle"
+
+# Template to be used to generate a link for the release
+# __RELEASE_CYCLE__ will be replaced by the value of releaseCycle
+# __CYCLE_SHORT_HAND__ will be replaced by the optional changelogTemplate
+# __LATEST__ will be replaced by the value of latest
+# __LATEST_SHORT_HAND__ will be replaced by the optional latestShortHand
+# __CODENAME__ will be replaced by the optional codename
+
+# You can even use Liquid Templating inside the template, such as:
+# https://godotengine.org/article/maintenance-release-godot-__LATEST__
+# Do not use a localized URL (such as one containing en-us) if possible
+changelogTemplate: "https://link/of/the/__RELEASE_CYCLE__/and/__LATEST__/version"
+
+# Optional template that generates names for every release. Supports same templating as changelogTemplate.
+# Default value is `__RELEASE_CYCLE__``
+releaseLabel: "MoM Timeturner __RELEASE_CYCLE__ (__CODENAME__)"
+
+# The label that will be used alongside releases labelled with `lts: true`
+# Optional, only provide if the product has lts releases that are not called LTS, but something else.
+# Default Value is <abbr title='Long Term Support'>LTS</abbr>
+# Prefer using an HTML abbr tag, if possible.
+LTSLabel: "<abbr title='Extra Long Support'>ELS</abbr>"
+
+# Optional information about how release information can be fetched automatically
+# This is mainly used for the `latest` and `latestDate` fields of each release cycle
+# Please see https://github.com/endoflife-date/endoflife.date/wiki/Automation for more details
+# This is currently a WIP, and will not have any impact, but is highly recommended if this can be made available.
+auto:
```

Comment:
If you will not use auto parts just remove all these parts because even git address doesnt exist and i do believe
devices generally doesnt have a github repo